packages/modern-browsers/modern.js

```diff
@@ -30,7 +30,7 @@ const browserAliases = {
 
   // The webapp package converts browser names to camel case, so
   // mobile_safari and mobileSafari should be synonymous.
-  mobile_safari: ["mobileSafari"],
+  mobile_safari: ["mobileSafari", "mobileSafariUI", "mobileSafariUI/WKWebView"],
```

Comment:
https://github.com/meteor/meteor/blob/ffcfa5062cf1bf8a64ea64fef681ffcd99fe7939/packages/meteor/cordova_environment.js#L14

Are you sure this works for cordova? I saw this code and it might seem we are defaulting to legacy always on cordova.

Reply:
Yes, that is what I thought, but for @schlaegerz this worked:
```js
setMinimumBrowserVersions(
    {
      "mobileSafariUI/WKWebView": 10,
    },
    "classes"
  );
```
And so did this PR. I think this is not related to Cordova as from my understanding. More like a Meteor app being displayed through some other app instead of being build via Cordova (I have seen this done in the past).

Here is an article about the different views:
https://developer.apple.com/news/?id=trjs0tcd

Reply:
Yes that is correct this is not Cordova, for various reasons we end up hosting a webview inside of our app that points to our Meteor app. It is for sure a weird edge case that is not too common, but this change definitely matches reality more since the WKWebview is completely a modern browser.